examples/spec/cassandra.ini

```diff
@@ -0,0 +1,1978 @@
+[]
+mountpoint = cassandra.ini
+infos/plugins = dump
+
+
+[/cluster_name]
+;type = string
```

Comment:
Why?

Reply:
From the Pull Request Description:
> Please note that some checks are intentionally commented out because of the max plugin error.
With this they can at least still be spec-mounted.

Reply:
As said in the PR description, such information should be within the repo, not in Pull Request Descriptions.